### DISCUSSION TITLE
tquic是否有内置的对应监控的metrics

### DISCUSSION THREAD

**Discussion by zelot1982** [Tencent/tquic#367] [General]
tquic是否有内置的对应监控的metrics

**Comment by iyangsj** [comment]
可以通过相关接口获取QUIC连接/路径级别的统计指标

C/C++语言接口：
https://tquic.net/zh/docs/api_reference/c_quic/#quic_conn_path_stats
https://tquic.net/zh/docs/api_reference/c_quic/#quic_conn_stats

Rust 语言接口：
https://docs.rs/tquic/latest/tquic/connection/struct.Connection.html#method.get_path_stats
https://docs.rs/tquic/latest/tquic/connection/struct.Connection.html#method.stats
